music.go

```diff
@@ -0,0 +1,53 @@
+package pctk
+
+import (
+	"log"
+	"os"
+	"unsafe"
+
+	rl "github.com/gen2brain/raylib-go/raylib"
+)
+
+// Music is music and we love it!
+type Music = rl.Music
+
+func (a *App) LoadMusic(fileName string) {
+	if _, err := os.Stat(fileName); err != nil {
+		log.Fatalf("An error occurred while attempting to load the music file %s: %v", fileName, err)
+	}
+
+	// dispose the previous music if any
+	if unsafe.Pointer(&a.music) != nil {
```

Comment:
This will never evaluate to false. Please note you are testing that `&a.music` is nil here. It is a pointer to a field of the app struct, so it will never be nil. 

If you want to check if the music is loaded or not, there is a `rl.IsMusicReady()` function that can be used for that.

Reply:
You are right! Thanks

Reply:
Done